app/controllers/AMIable.scala

```diff
@@ -46,7 +46,10 @@ class AMIable @Inject()(override val amiableConfigProvider: AmiableConfigProvide
         amiWithUpgrade = Recommendations.amiWithUpgrade(agents.allAmis)(ami)
         instances <- Prism.imageUsage(ami)
         launchConfigs <- Prism.launchConfigUsage(ami)
-      } yield Ok(views.html.ami(amiWithUpgrade, instances, launchConfigs))
+      } yield Ok(views.html.ami(
+        amiWithUpgrade,
+        PrismLogic.sortInstancesByStack(instances),
+        PrismLogic.sortLCsByOwner(launchConfigs)))
```

Comment:
Would it make sense to define an `Ordering` in the companion object of `Instance` and `LaunchConfiguration` and then use `.sorted`